Soccar [UI]/src/nl/soccar/ui/fx/controller/SessionViewFXMLController.java

```diff
@@ -136,9 +130,8 @@ private void leaveRoom() {
         ClientController controller = ClientController.getInstance();
         Connection connection = controller.getCurrentConnection();
         Room room = currentPlayer.getCurrentSession().getRoom();

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "room" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "room". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1854)